dids/didweb/didweb.go

```diff
@@ -173,7 +173,7 @@ func Create(domain string, opts ...CreateOption) (_did.BearerDID, error) {
 	}
 
 	for _, svc := range options.services {
-		document.AddService(&svc)
+		document.AddService(&svc) //nolint:gosec
```

Comment:
this one is the only one which concerns me that I shouldn't ignore...

Reply:
my initial reaction was `1.22` fixed this and that the `gosec` linter was behind

I guess we could really beef up the test you added in that PR, but idk seems like a non-issue in the current context

Reply:
Yep 1.22 fixed this.